tests/automation_framework/data/worker/worker_update.ini

```diff
@@ -0,0 +1,24 @@
+[DEFAULT]
+method=WorkerUpdate
+id=2
+params:{
+            "workerId":"",
+            "details":
+            {
+                "hashingAlgorithm": "KECCAK-256"
+            }
+        }
+
+[18266]
+workerId=4582
```

Comment:
Same as above, workerId should be populated automatically based on the code in worker lookup and worker retrieve params file.

the test name is test_worker_update_unknown_parameter, then why are we removing the hashingalgorithm parameter here?